src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import * as topojson from "topojson-client";
+  import * as Plot from "@observablehq/plot";
+
+  export let data;
+  $: ({ fireWeatherData } = data);
+  $: parishGeodata = fireWeatherData?.topoJSON ?? [];
+  $: fireData = fireWeatherData?.parishes ?? [];
+  $: parishes = topojson.feature(
+    parishGeodata,
+    parishGeodata.objects["parishes-fullsize"],
+  ).features;
+
+  $: if (parishes) {
+    parishes.forEach((parish: any) => {
+      const name = parish.properties.parishname.replaceAll(".", "");
+      const match = fireData.find((parish: any) => parish.ParishName === name);
+      parish.properties.fireRisk = match?.DangerClassDay || 0;
+    });
+  }
+
+  // TODO: understand difference between topojson.feature vs topojson.mesh
+  // const louisiana = topojson.mesh(parishData,parishData.objects['parishes-fullsize']);
+
+  let div: HTMLElement;
+
+  const fireRiskLevels = ["Low", "Medium", "High", "Very High", "Extreme"];
+  const colors = [
+    // "#D0D0D0", Omitted here because we don't want the 'null' value to be included in the legend
```

Comment:
I wonder if we should equate `null` with "None"? Might be something to ask folks in the department. As it stands now it could be unclear whether the greyed-out parishes have no risk or _no data_.

Reply:
That's a good point to get clarity on. In the sample data they sent over, there were 12/64 parishes that are simply not included. To your point, no data _could_ be equivalent to no risk, but that was my assumption and not a given.